src/functions/module_lookup.ts

```diff
@@ -829,7 +830,8 @@ export const PIVOT = {
             result[col].push({ value: "" });
             break;
           case "HEADER":
-            result[col].push(pivot.getPivotHeaderValueAndFormat(pivotCell.domain));
+            const value = pivot.getPivotHeaderValueAndFormat(pivotCell.domain);
```

Comment:
value => result ?
`value` makes me think it's only the value, but it's actually the object with the format

Reply:
`result` is already used in this scope. Repalced by `valueAndFormat`